### ISSUE TITLE
Done tasks marked by Time Ruler are marked with time

### ISSUE THREAD

**Issue by Djyp** [j-palindrome/obsidian-time-ruler#114]
When I click on a task to mark it done, the correct emoji and date appear at the end of my task BUT there is also a time mark in the format `Thh:mm` at the end of it (2024-04-19T16:54). As the author of the task plugin says, any time information behind the emojis will break how the plugin reads them.

Therefore, this tasks code block :
````
```tasks
done 2024-04-19
```
````
will show no task done today.

There is a workaround, meanwhile, with dataview :
````
```dataview
TASK
WHERE completed and contains(text, "✅ 2024-04-19")
```
````

Possible solution :
Maybe if the user selected the Tasks format, a done task should had a only a date and no time

**Comment by j-palindrome** [comment]
This is true—I can add a toggle to disable this. Added to changelog

**Comment by j-palindrome** [comment]
This has been fixed in v2.5.0.
